src/Components/DataFetchDemo.tsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+export const DataFetchDemo = () => {
```

Comment:
Something is fishy with the typing in this. The axios response is typed as a string, but we also assume its an array. My editor shows errors anyway